omod/src/main/java/org/openmrs/module/reporting/web/reports/renderers/ExcelReportRendererFormController.java

```diff
@@ -213,5 +215,9 @@ else if (successUrl.startsWith(pathToRemove)) {
     	}
     	design = rs.saveReportDesign(design);
     	return "redirect:" + successUrl;
+       }catch(PropertyValueException e) {
```

Comment:
Same applies here and anywhere else in this PR

Reply:
I did several testing of that , and instead of a Null Pointer Exception being throen  when an empty parameter is passed to the controller, its a PropertyValueEception that is thrown . i think the NPE is already manged by the util methods used there in that specific controller where i was catching the propertValueException